libs/types/src/cdf/cast-vote-records/vx-schema.json

```diff
@@ -196,7 +196,7 @@
       "type": "object"
     },
     "CVR.CVRContest": {
-      "required": ["@type", "ContestId"],
+      "required": ["@type", "ContestId", "CVRContestSelection"],
```

Comment:
Rationale: this just made the typing easier. Use an empty array if there are no selections. Can revert this one if it seems excessive.

Reply:
I'm into this. I think in general empty array communicates "there were no selections" better than undefined, which sounds more like "we don't know if there were selections"

Reply:
OK, I wasn't sure if making changes to our spec for typing purposes was overkill, because we technically are restricting the interoperability of the format. But for the CVR project we've never been terribly concerned about robust interoperability, so I thought it better to make adjustments for ourselves.

Reply:
I think it's useful to be as constrained as possible until we need more flexibility